spec/services/refresh_reporting_views_spec.rb

```diff
@@ -66,9 +66,9 @@
         create(:blood_pressure, patient: patient, recorded_at: 2.days.ago)
         create(:appointment, patient: patient, recorded_at: 1.day.ago)
 
-        RefreshReportingViews.call(views: ["Reports::DailyFollowUp"])
+        RefreshReportingViews.call(views: ["Reports::FacilityDailyFollowUpAndRegistration"])
       end
-    }.to change { Reports::DailyFollowUp.count }.by(2)
+    }.to change { Reports::FacilityDailyFollowUpAndRegistration.count }.to be > 0
```

Comment:
Why not a specific number?

Reply:
This number will be `number of facilities * 31` and this may vary. To check if the views are refreshed it need not be a specific number rt?